docs/index.html

```
@@ -527,5 +711,118 @@ <h2>Open Source & Free</h2>
+            0%, 100% { transform: scaleY(1); }
+            50% { transform: scaleY(1.22); }
+        }
+
+        @media (prefers-reduced-motion: reduce) {
+            .string {
+                animation: none;
+            }
+        }
+    </style>
+        }
+    </style>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </style> ]

(tag-pair)